<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>996</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>996</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>基于图像处理的人脸检测</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 italic 11pt &quot;Ubuntu Mono&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="rgbLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="gradLabel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="grayLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="segLabel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>641</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>141</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>141</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>640</width>
       <height>140</height>
      </rect>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>156</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>35</number>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>996</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevice">
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionCamera"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <addaction name="menuDevice"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background: lightgray;</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCamera"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionClose"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_6">
   <property name="minimumSize">
    <size>
     <width>290</width>
     <height>650</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>290</width>
     <height>661</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">::title {
      text-align: left; 
      background: lightgray;
      padding-left: 5px;
  }

::close-button, ::float-button {
      border: 1px solid transparent;
      background: darkgray;
      padding: 0px;
}

::close-button:hover, QDockWidget::float-button:hover {
      background: gray;
  }

::close-button:pressed, ::float-button:pressed {
      padding: 1px -1px -1px 1px;
  }






</string>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <property name="styleSheet">
     <string notr="true">border:1px solid;
border-color: rgb(186, 189, 182);
border-bottom:0px;</string>
    </property>
    <widget class="MyComboBox" name="widget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>290</width>
       <height>641</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionCamera">
   <property name="icon">
    <iconset resource="qrsc.qrc">
     <normaloff>:/img/相机.svg</normaloff>:/img/相机.svg</iconset>
   </property>
   <property name="text">
    <string>Cam</string>
   </property>
   <property name="toolTip">
    <string>设备</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="qrsc.qrc">
     <normaloff>:/img/刷新.svg</normaloff>:/img/刷新.svg</iconset>
   </property>
   <property name="text">
    <string>Ref</string>
   </property>
   <property name="toolTip">
    <string>刷新</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="qrsc.qrc">
     <normaloff>:/img/关闭.svg</normaloff>:/img/关闭.svg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <zorder>dockWidget_6</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QWidget</extends>
   <header>mycombobox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qrsc.qrc"/>
 </resources>
 <connections/>
</ui>
